go/vt/vtgate/plugin_mysql_server.go

```diff
@@ -127,6 +129,12 @@ func initMySQLProtocol() {
 		return
 	}
 
+	// Check for the connection threshold
+	if mysqlSlowConnectWarnThreshold.Nanoseconds() != 0 {
```

Comment:
Of course, this will probably just be:
mysqlListener.SlowConnectWarnThreshold = *mysqlSlowConnectWarnThreshold
once you change it to be on the Listener. See AllowClearTextWithoutTLS in this file.